src/main/java/hadouken/SimpleMessage.java

```diff
@@ -56,6 +57,7 @@ public JsonNode getJsonBody() throws IOException {
   }
 
   public org.codehaus.jackson.JsonNode getXmlBody() throws IOException {
+    // NOTE(justin.morgan): Why do we have two implementations of jackson.JsonNode? o_O
```

Comment:
@jlmorgan `jackson.JsonNode` is what we use for JSON, and yet `org.codehaus.jackson.JsonNode` is what comes back from `com.fasterxml.jackson.xml.XmlMapper.readTree`, for reasons that baffle me. Could we maybe make a note of that instead of leaving the question hanging as a comment?